ocfweb/docs/templates/commands.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div class="ocf-content-block">
+    <h2>Shell interface</h2>
+    <div class="shell-wrapper">
+      <iframe src="https://ssh.ocf.berkeley.edu" class="shell-frame">
+        <p>Sorry! Your browser can't display this window.</p>
+      </iframe>
+    </div>
+    <p>
+      Welcome to the live command reference! Here you can securely access our
+      login server via <a href="{% url 'doc' 'services/shell' %}">SSH</a> and
+      run commands to manage your account, edit a website, or work on our
+      server. Below is a list of account management commands with a brief
+      description of what they do and how to use them. Simply type in a command
+      and hit enter to run it. To log out, run the command <code>exit</code>.
+    </p>
+    <h2>OCF commands</h2>
+      <table class="table table-striped table-hover">
+        <caption>
+          <strong>Key:</strong>
+          <code>ARGUMENT</code>
+          <code>[optional]</code>
+        </caption>
+        <tr>
+          <th>Command</th>
+          <th>Description</th>
+        </tr>
+        {% for command in ocf_commands %}
+          <tr>
+            <td>
+              <code>
+                <strong>
+                  {% if command.doc %}
+                    <a href="{% url 'doc' command.doc %}#{{command.doc_anchor}}">
+                      {{command.name}}
+                    </a>
+                  {% else %}
+                    {{command.name}}
+                  {% endif %}
+                </strong>
+                {% if command.args %}
+                  {{command.args}}
+                {% endif %}
+              </code>
+            </td>
+            <td>
+              {{command.desc | safe}}
+            </td>
+          </tr>
+        {% endfor %}
+      </table>
+    <h2>File commands</h2>
+      <p>
+        For convenience, here is a very basic listing of commands to manage
+        files. For a more complete listing, see for example <a
+        href="https://en.wikipedia.org/wiki/List_of_Unix_commands">
+        Wikipedia</a>. For more information on a specific command, run
+        <code>man COMMAND</code>.
+      </p>
+      <table class="table table-striped table-hover">
+        <caption>
+          <strong>Key:</strong>
+          <code>ARGUMENT</code>
+          <code>[optional]</code>
+        </caption>
+        <tr>
+          <th>Command</th>
+          <th>Description</th>
+        </tr>
+        {% for command in file_commands %}
+          <tr>
+            <td>
+              <code>
+                <strong>
+                  {% if command.doc %}
+                    <a href="{% url 'doc' command.doc %}">{{command.name}}</a>
+                  {% else %}
+                    {{command.name}}
+                  {% endif %}
+                </strong>
+                {{command.args}}
+              </code>
+            </td>
+            <td>
+              {{command.desc}}
```

Comment:
Culprit of the formatting bug; it should be `{{command.desc | safe}}` as it is above. This is why it should be a partial template if we keep it.